mypy_requirements.txt

```diff
@@ -0,0 +1,2 @@
+mypy==0.902
+types-PyYAML==0.1.6
```

Comment:
Can you create mypy_requirements.in from which we will then generate this file?

Reply:
Discussed on Hangouts. No need of this.